code/game/gamemodes/cult/cult.dm

```diff
@@ -167,21 +166,11 @@
 	to_chat(cult_mob, "\red [pick("You remember something from the dark teachings of your master","You hear a dark voice on the wind","Black blood oozes into your vision and forms into symbols","You have a vision of a [pick("crow","raven","vulture","parrot")] it squawks","You catch a brief glimmer of the otherside")]... [wordexp]")

```

Comment:
Not too hard if we can define that within the datums. Else, I'd be willing to write those